source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -376,6 +384,24 @@ namespace PlayFab
             playFabEmitRequest->stdCallback(playFabEmitRequest->event, response);
         }
     }
+
+    bool PlayFabEventPipeline::TryGetBatchOutOfFlight(void* customData, std::vector<std::shared_ptr<const IPlayFabEmitEventRequest>>* batchReturn)
+    {
+        // LOCK batchesInFlight mutex
+        std::unique_lock<std::mutex> lock(inFlightMutex);
+        auto iter = this->batchesInFlight.find(customData);
+        if (iter == this->batchesInFlight.end())
+        {
+            // not finding the batch in the queue is a bug
```

Comment:
I think having the `// not find the batch in the queue is a bug` and `// normal this never happens` comments below is redundant. I would opt for keeping the former comment and removing the latter comment since the first comment is clear that something is a bug rather than uncommon but expected behavior.